packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -180,6 +180,30 @@ export function ArticleEditor({id}: ArticleEditorProps) {
     }
   }, [createError, updateError, publishError])
 
+  function countRichTextBlocksChars(blocks: any) {
+    return blocks
+      .filter((value: any) => value.type === BlockType.RichText)
```

Comment:
Don't forget to count the Text length of the `BlockType.Title`. The title and lead also count to that number